Templates/ServiceBusTopicTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "ServiceBusTopicTrigger - C#",
+    "defaultFunctionName": "ServiceBusTopicTriggerCSharp",
+    "description": "A C# function that will be run whenever a message is added to a specified Service Bus topic",
+    "language": "C#",
+    "category": [
+        "Core",
+        "Data Processing"
+    ],
+    "userPrompt": [
+        "connection",
+        "accessRights",
+        "topicName"
```

Comment:
Yes i tested it out end to end. It worked because all mandatory settings are populated for the user from the default value in binding.json.